src/lwc/geBatchGiftEntryHeader/__tests__/geBatchGiftEntryHeader.test.js

```diff
@@ -76,6 +78,8 @@ describe('c-ge-batch-gift-entry-header', () => {
             getGiftBatchTotalsBy.mockResolvedValue(APEX_GIFT_BATCH_TOTALS_BY_SUCCESS);
```

Comment:
These `mockResolvedValue` can be removed since we're no longer importing that apex method in this component.

Instead, we'll need to populate the public `batchTotals` property with the values needed to verify that the elements this jest tests expects gets rendered appropriately.

Something like below perhaps
```
element.batchTotals = {
    hasValuesGreaterThanZero: true,
    totalGifts: 20,
    processedGifts: 10,
    failedPayments: 0,
    failedGifts: 5,
    expiredPayments: 0
}
```

Reply:
above keys are wrong
```
element.batchTotals = {
                hasValuesGreaterThanZero: true,
                totalGiftsCount: 20,
                processedGiftsCount: 10,
                failedPaymentsCount: 0,
                failedGiftsCount: 5,
                hasPaymentsWithExpiredAuthorizations: false
            }
```